pkg/beeclient/debugapi/postage.go

```diff
@@ -16,8 +16,15 @@ type postageResponse struct {
 }
 
 type PostageStampResponse struct {
-	BatchID     string `json:"batchID"`
-	Utilization uint32 `json:"utilization"`
+	BatchID       string         `json:"batchID"`
+	Utilization   uint32         `json:"utilization"`
+	Usable        bool           `json:"usable"`
+	Label         string         `json:"label"`
+	Depth         uint8          `json:"depth"`
+	Amount        *bigint.BigInt `json:"amount"`
+	BucketDepth   uint8          `json:"bucketDepth"`
+	BlockNumber   uint64         `json:"blockNumber"`
+	ImmutableFlag bool           `json:"immutableFlag"`
```

Comment:
There were added several fields recently, such as `Exists` and `BatchTTL`.

Reply:
done :+1: